R/expr__string.R

```diff
@@ -12,14 +12,15 @@
 #' When parsing a Datetime the column precision will be inferred from the format
 #' string, if given, e.g.: `"%F %T%.3f"` => [`pl$Datetime("ms")`][pl_Datetime].
 #' If no fractional second component is found then the default is `"us"` (microsecond).
-#' @param datatype The data type to convert into. Can be either Date, Datetime,
-#' or Time.
+#' @param dtype The data type to convert into. Can be either `pl$Date`,
+#' `pl$Datetime()`, or `pl$Time`.
```

Comment:
Please link to the document.

Reply:
Actually we only have docs for `pl$Datetime()`, not the others (`?pl_Date` links to `pl$date()`, which is not what we want here)